patches/0005-Modify-resources.patch

```diff
@@ -2174,20 +2174,20 @@ index e4230415f8..5dabd44191 100644
  						},
  
  						"resource_id": {
-@@ -180,7 +183,8 @@ func resourceSecurityCenterAutomation() *pluginsdk.Resource {
- 														string(security.LesserThanOrEqualTo),
- 														string(security.NotEquals),
- 														string(security.StartsWith),
+@@ -185,7 +188,8 @@ func resourceSecurityCenterAutomation() *pluginsdk.Resource {
+ 														string(automations.OperatorLesserThanOrEqualTo),
+ 														string(automations.OperatorNotEquals),
+ 														string(automations.OperatorStartsWith),
 -													}, false),
 +													}, features.CaseInsensitive()),
 +													DiffSuppressFunc: suppress.CaseDifferenceV2Only,
  												},
  												"property_type": {
  													Type:     pluginsdk.TypeString,
-@@ -190,7 +194,8 @@ func resourceSecurityCenterAutomation() *pluginsdk.Resource {
- 														string(security.String),
- 														string(security.Boolean),
- 														string(security.Number),
+@@ -195,7 +199,8 @@ func resourceSecurityCenterAutomation() *pluginsdk.Resource {
+ 														string(automations.PropertyTypeString),
+ 														string(automations.PropertyTypeBoolean),
+ 														string(automations.PropertyTypeNumber),
```

Comment:
Do we know the reasons for this change? Can we link or add a description? Might be helpful when reconciliating patches in the future.

Reply:
It's kind of confusing, since we are looking at a patch of a patch. That is actually a change in the context of our patch change, not the patch change itself. This is all upstream.


Looking at the new patch without the diff might clarify:

```patch
@@ -185,7 +188,8 @@ func resourceSecurityCenterAutomation() *pluginsdk.Resource {
 														string(automations.OperatorLesserThanOrEqualTo),
 														string(automations.OperatorNotEquals),
 														string(automations.OperatorStartsWith),
-													}, false),
+													}, features.CaseInsensitive()),
+													DiffSuppressFunc: suppress.CaseDifferenceV2Only,
 												},
 												"property_type": {
 													Type:     pluginsdk.TypeString,
@@ -195,7 +199,8 @@ func resourceSecurityCenterAutomation() *pluginsdk.Resource {
 														string(automations.PropertyTypeString),
 														string(automations.PropertyTypeBoolean),
 														string(automations.PropertyTypeNumber),
-													}, false),
+													}, features.CaseInsensitive()),
+													DiffSuppressFunc: suppress.CaseDifferenceV2Only,
 												},
 											},
 										},
```